OA-PASS/pass-docker
# Support context 3.4

* Bakes context 3.4 into fcrepo image
* Bumps indexer to version 0.0.18
* Reindexed existing es index using newest esconfig, due to backwards-incompatible [normalizer change](https://github.com/OA-PASS/pass-data-model/commit/108609265cf33aeb7c6e9149564017c29d537c58)

## To test

     curl -X POST -H "Content-Type: application/ld+json" -H "Slug: foo" -d '{
        "@context": "https://oa-pass.github.io/pass-data-model/src/main/resources/context-3.4.jsonld",
        "@id": "",
        "@type": "Submission",
        "effectivePolicies": "http://localhost:8080/fcrepo/rest/policies/3b/73/9a/e7/3b739ae7-a394-4001-98fa-bb45bfabf792"
    }' -u "fedoraAdmin:moo" http://localhost:8080/fcrepo/rest/submissions

That should create a submission named `http://localhost:8080/fcrepo/rest/submissions/foo`.  Do an elasticsearch query for it, and verify that effectivePolicies was indexed:

    http://localhost:9200/pass/_search?q=@id:"http://fcrepo:8080/fcrepo/rest/submissions/foo"&pretty

You can also search for that submission by effective policy:

    http://localhost:9200/pass/_search?q=effectivePolicies:%22http://fcrepo:8080/fcrepo/rest/policies/3b/73/9a/e7/3b739ae7-a394-4001-98fa-bb45bfabf792%22&pretty

Resolves #191, OA-PASS/pass-data-model#56
